src/prelaunch/jungle.hunt.a

```diff
@@ -0,0 +1,27 @@
+;license:MIT
+;(c) 2019 by qkumba
+
+!cpu 6502
+!to "build/PRELAUNCH/JUNGLE.HUNT",plain
+*=$106
+
+    !source "src/prelaunch/common.a"
+
+         lda   #$60
+         sta   $81E
+         jsr   $800
+
+         lda   #$60
+         sta   $BD44
+         jsr   $BD00
+
+         lda   #1
+         sta   $0A6C
+         lda   #0
+         sta   $0A67 ; reset vector patch
```

Comment:
Does this work on all levels? We had problems creating cheats because so much code is reloaded when it switches from vines to underwater to rocks, &c.

Reply:
I'm patching an LDA # STA $3F2/3F3 combo here so it seems unlikely it would be reused, but I can try playing through for a while to verify. A=0 at the patch point, which is why the "1" is set first.

Reply:
yah seems fine